src/lib.rs

```diff
@@ -61,7 +61,9 @@ pub struct Response {
     /// An error if there is one, or null
     pub error: Option<error::RpcError>,
     /// Identifier from the request
-    pub id: json::Value
+    pub id: json::Value,
+    /// jsonrpc field, MUST be "2.0"
+    pub jsonrpc: json::Value
```

Comment:
My feeling is that it's fine if it's mandatory on the request. JSONRPC 1 clients will just ignore the extra field.